packages/data-broker/src/services/DataServiceMapping.js

```diff
@@ -70,4 +72,28 @@ export class DataServiceMapping {
     }
     return null;
   }
+
+  /**
+   * @param {DataServiceMapping} other
+   * @returns boolean
+   */
+  equals(other) {
```

Comment:
This warms an ex-Java dev's heart 😄 Bonus points for adding `if (this === other) return true;` at the start 👍